src/main/java/com/ncedu/fooddelivery/api/v1/dto/order/CountOrderCostResponseDTO.java

```diff
@@ -0,0 +1,16 @@
+package com.ncedu.fooddelivery.api.v1.dto.order;
+
+import lombok.Data;
+
```

Comment:
У lombok есть крутая аннотация.
```suggestion
@AllArgsConstructor
```